### DISCUSSION TITLE
How does mpv remember the position in a playlist ?

### DISCUSSION THREAD

**Discussion by sebma** [mpv-player/mpv#14740] [Q&A]
Hi, I am using mpv v0.38 and I accidentally run mpv 2 or 3 times and somehow ended up losing the position of mpv in my playlist in mpv watch_later files .

So I tried [this](https://github.com/mpv-player/mpv/issues/7637#issuecomment-621949527) to remove the watch_later MD5 of the first video in my playlist but it did not work.

How does mpv remember the position in a playlist ?

**Comment by guidocella** [comment]
mpv remembers the position by creating a text file named like the MD5 hash of the file path. Why would removing these files restore the position?

**Comment by sebma** [comment]
I meant the index in the playlist, sorry, I used the wrong word.

The MD5 file seems to store the position in a video, how does mpv remember the index in a playlist ?

**Reply by guidocella** [reply]
They are remembered in the same way. If a watch later file for a playlist entry exists, playback resumes from that file.

**Reply by sebma** [reply]
@guidocella And if more than one watch later file for a playlist exist, how does `mpv` decide which file to resume from ?

**Reply by guidocella** [reply]
It resumes from the first playlist entry with a watch later file.
